### PR TITLE
- Created md5 method in Service class wich hash the passwords

### PR DESCRIPTION
**Pull request by UrsuVasile**

### CODE REVIEW

File: src/main/java/Service/Service.java [hunk 1]
```diff
@@ -217,4 +223,50 @@ private void findProductByIdCommand() {
...
+
+            //Update input string in message digest
+            digest.update(input.getBytes(), 0, input.length());
+
+            //Converts message digest value in base 16 (hex)
+            md5 = new BigInteger(1, digest.digest()).toString(16);
+
+        } catch (NoSuchAlgorithmException e) {
+
+            e.printStackTrace();
```
Comment (stefan2212) [inline src/main/java/Service/Service.java]: Never leave stacktraces like this. Also putna sugestive mesaage near it

File: src/main/java/Service/Service.java [hunk 2]
```diff
@@ -217,4 +223,50 @@ private void findProductByIdCommand() {
...
+            byte[] hashedBytes = sha.digest(input.getBytes());
+            char[] digits = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
+                    'a', 'b', 'c', 'd', 'e', 'f'};
+            for (int idx = 0; idx < hashedBytes.length; idx++) {
+                byte b = hashedBytes[idx];
+                hash.append(digits[(b & 0xf0) >> 4]);
+                hash.append(digits[b & 0x0f]);
+            }
+        } catch (NoSuchAlgorithmException e) {
+            // handle error here.
```
Comment (stefan2212) [inline src/main/java/Service/Service.java]: Also give a message here

**Review by stefan2212 — CHANGES_REQUESTED** [review]

File: src/main/java/Service/Service.java (hunk 1 above)
Comment (UrsuVasile) [reply src/main/java/Service/Service.java]: solved

File: src/main/java/Service/Service.java (hunk 2 above)
Comment (UrsuVasile) [reply src/main/java/Service/Service.java]: solved

Comment (UrsuVasile) [reply src/main/java/Service/Service.java]: is it ok or there is more problems?

**Pull request merged**